src/main/webapp/search.js

```diff
@@ -78,14 +78,14 @@ function addVideoInfo(video) {
   cardBodyDiv.className = 'card-body';
   cardBodyDiv.innerHTML = '<p class="card-text">' +
     video.snippet.description + '</p';
-  cardDiv.innerHTML = '<IMG class="card-img-top" src=' +
-    video.snippet.thumbnails.high.url + ' alt="Card image cap">';
+  cardDiv.innerHTML = '<iframe width="480" height="315"src=' + 'https://www.youtube.com/embed/'+video.id.videoId+ '?controls=0;' +
```

Comment:
Is it possible for us to move the height and width values to css?

Reply:
Hey cis, I tried multiple ways doing that. It looks like the iframe's width and height can only be defined only when it has been created. We are creating the element here and so it is much more complicated if we want to move it into css.